### PR TITLE
ensure equivalent Guardian perm exists during data migration

### PR DESCRIPTION
**Pull request by escattone**
https://github.com/mozilla/sumo/issues/1144

### CODE REVIEW

File: kitsune/forums/migrations/0003_authority_to_guardian_data.py [hunk 1]
```diff
@@ -1,20 +1,53 @@
...
-    "forums_forum.thread_edit_forum": "forums.edit_forum_thread",
-    "forums_forum.thread_delete_forum": "forums.delete_forum_thread",
-    "forums_forum.thread_move_forum": "forums.move_forum_thread",
-    "forums_forum.thread_sticky_forum": "forums.sticky_forum_thread",
-    "forums_forum.thread_locked_forum": "forums.lock_forum_thread",
-    "forums_forum.post_edit_forum": "forums.edit_forum_thread_post",
-    "forums_forum.post_delete_forum": "forums.delete_forum_thread_post",
-    "forums_forum.post_in_forum": "forums.post_in_forum",
-    "forums_forum.view_in_forum": "forums.view_in_forum",
+    "forums_forum.thread_edit_forum": ("edit_forum_thread", "Can edit forum threads"),
```
Comment (akatsoulas) [inline kitsune/forums/migrations/0003_authority_to_guardian_data.py:7]: Let's add a trailing comma to the tuples

File: kitsune/forums/migrations/0003_authority_to_guardian_data.py [hunk 2]
```diff
@@ -1,20 +1,53 @@
...
 }
 
 
+def get_or_create_guardian_perm(authority_perm):
+    """
+    Given an authority.models.Permission instance, returns its equivalent Django
+    Permission (django.contrib.auth.models.Permission) instance. If the equivalent
+    Django Permission instance doesn't already exist, it is created.
+    """
+    assert authority_perm.codename in AUTHORITY_TO_GUARDIAN
```
Comment (akatsoulas) [inline kitsune/forums/migrations/0003_authority_to_guardian_data.py:40]: Shall we just silently ignore anything not in the dictionary and just potentially create the perms that exist in there?

**Review by akatsoulas — APPROVED** [review]
r+wc :+1:

File: kitsune/forums/migrations/0003_authority_to_guardian_data.py (hunk 2 above)
Comment (escattone) [reply kitsune/forums/migrations/0003_authority_to_guardian_data.py:40]: Yes, sounds good, let's do that.

**Pull request merged**